src/picknik_ur_site_config/objectives/getdoorgraspableobjectsubtree.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="GetDoorGraspableObjectSubtree">
```

Comment:
I think you can omit the "Subtree" from the sames of all of these, so `GetDoorGraspableObject`.

Also, for consistent styling, name the files with underscores, so `get_door_graspable_object.xml`.

And same for the remaining ones.

Reply:
Fixed. This is a problem with our `create Objective` utility not following our internal naming convention.